Docker-Swarm-deployment/single-node/immudb/docker/immudb-config-generator/immudb-config-generator.py

```diff
@@ -21,37 +23,44 @@
     except:
         continue
 
-client = ImmudbClient("immudb:3322")
+client = ImmudbClient("{0}:3322".format(Host))
 client.login("immudb","immudb")
 
 client.changePassword("immudb",ADMIN_PASSWORD,"immudb")
 client.login("immudb",ADMIN_PASSWORD)
 
-client.databaseCreate("iudxauth")
-client.databaseCreate("iudxcat")
-client.databaseCreate("iudxrs")
+client.createDatabase("iudxauth")
+client.createDatabase("iudxcat")
+client.createDatabase("iudxrsorg")
 
 client.createUser("iudx_auth",IUDX_AUTH_PASSWORD,immudb.constants.PERMISSION_RW,"iudxauth")
 client.createUser("iudx_cat",IUDX_CAT_PASSWORD,immudb.constants.PERMISSION_RW,"iudxcat")
-client.createUser("iudx_rs",IUDX_RS_PASSWORD,immudb.constants.PERMISSION_RW,"iudxrs")
+client.createUser("iudx_rs",IUDX_RS_PASSWORD,immudb.constants.PERMISSION_RW,"iudxrsorg")
 
-client.databaseUse("iudxcat")
-client.sqlExec("CREATE TABLE auditingtable(id VARCHAR NOT NULL, userRole VARCHAR NOT NULL,userID VARCHAR NOT NULL,iid VARCHAR NOT NULL,api VARCHAR NOT NULL,method VARCHAR NOT NULL,time INTEGER NOT NULL,iudxID VARCHAR NOT NULL,PRIMARY KEY id);")
+client.useDatabase("iudxcat")
+client.sqlExec("CREATE TABLE auditingtable(id VARCHAR[250] NOT NULL, userRole VARCHAR[250] NOT NULL,userID VARCHAR[250] NOT NULL,iid VARCHAR[250] NOT NULL,api VARCHAR[250] NOT NULL,method VARCHAR[250] NOT NULL,time INTEGER NOT NULL,iudxID VARCHAR[250] NOT NULL,PRIMARY KEY id);")
+
+client.useDatabase("iudxauth")
+client.sqlExec("CREATE TABLE table_auditing(id VARCHAR[250] NOT NULL, body VARCHAR[250] NOT NULL,userid VARCHAR[250] NOT NULL,endpoint VARCHAR[250] NOT NULL,method VARCHAR[250] NOT NULL,time INTEGER NOT NULL,PRIMARY KEY id);")
+
+client.useDatabase("iudxrsorg")
+client.sqlExec("CREATE TABLE IF NOT EXISTS rsauditingtable (id VARCHAR[128] NOT NULL, api VARCHAR[128], userid VARCHAR[128],epochtime INTEGER,resourceid VARCHAR[256], isotime VARCHAR[64], providerid VARCHAR[128], PRIMARY KEY id);")
+
+client.sqlExec("CREATE INDEX ON rsauditingtable(userid, epochtime, providerid, resourceid);")
+
+client.sqlExec("CREATE TABLE rsaudit (id VARCHAR[128] NOT NULL,api VARCHAR[128] NOT NULL,userid VARCHAR[128] NOT NULL,epochtime INTEGER NOT NULL,resourceid VARCHAR[256] NOT NULL,isotime VARCHAR[64] NOT NULL,providerid VARCHAR[128] NOT NULL,size INTEGER, PRIMARY KEY id);")
```

Comment:
Change size to responsesize

Reply:
The API servers are changed to insert as size. Lets not change the schema. We will keep it as size

Reply:
okay.